src/components/Vue2PortedPackages/VueImageUploader.vue

```diff
@@ -202,10 +205,10 @@ export default {
       this.$emit("change", output);
     },
     emitLoad() {
-      this.$emit("upload");
+      this.$emit("onLoad");
     },
     emitComplete() {
-      this.$emit("complete");
+      this.$emit("onComplete");
```

Comment:
this will become ononComplete like like mouseover becomes onmouseover. Any reason to change this? instead of staying consistent with others like `input`, `change` (and not `onInput`, `onChange`)